app/scripts/services/authorization.js

```diff
@@ -96,16 +96,22 @@ angular.module("openshiftConsole")
       return _.get(cachedRulesByProject.get(projectName || currentProject), ['rules']);
     };
 
+    var checkRulesForResourceVerb = function(rules, resource, verb) {
+      return _.contains(rules[resource], verb) || _.contains(rules[resource], '*');
+    };
+
     var canI = function(verb, resource, projectName) {
```

Comment:
to check resource=foo, apigroup=bar, you have to check four combinations to be accurate:
```
resource=foo, apigroup=bar
resource=foo, apigroup=*
resource=*, apigroup=bar
resource=*, apigroup=*
```
a data structure that facilitates that will also accommodate the first lookup, which is the only one we're currently doing